recipes/default.rb

```diff
@@ -36,6 +36,7 @@
 
 group 'scponly' do
   system true
+  not_if 'grep scponly /etc/passwd'
```

Comment:
So this shouldn't be here and this actually presented us with the fact that our ``scponly_user`` resource is also managing the ``scponly`` user. We need to fix the resource to properly deal with this likely by using ``edit_resource`` since we already manage this in the default recipe. Can you please do that?